ee/ui/assets/assets.go

```diff
@@ -0,0 +1,69 @@
+// DO NOT EDIT
+// THIS FILE IS AUTOGENERATED
+package assets
+
+import (
+	_ "embed"
+)
+
+
+//go:embed menubar-blocked-color-dark.ico
+var KolideMenuBlockedColorDarkIco []byte
```

Comment:
can't leave a comment on the pictures so, putting it here, I think some of the icons have dark / light switched

or maybe in the context of `menubar-will-block-color-light.png`, light means "light theme" as opposed tot he light color of the asset?

![image](https://user-images.githubusercontent.com/8415811/217566043-58f66213-9f94-4ab1-80f8-4f09f7d71c13.png)

Reply:
The current names describe the intended usage. But I think it would be more re-usable for them to describe the _icon_ and leave the usage up to the application and control layers. I need to circle back with Fritz and discuss that. It's probably the biggest reason this is still in draft.

But I like writing go generators, so I did that anyhow.